Keas.Mvc/Services/EventService.cs

```diff
@@ -23,6 +23,14 @@ public interface IEventService
         Task TrackAcceptKey(KeySerial keySerial);
         Task TrackAcceptEquipment(Equipment equipment);
         Task TrackAcceptWorkstation(Workstation workstation);
+        Task TrackKeyDeleted(Key key);
+        Task TrackAccessDeleted(Access access);
+        Task TrackEquipmentDeleted(Equipment equipment);
+        Task TrackWorkstationDeleted(Workstation workstation);
+        Task TrackAssignmentUpdatedKeySerial(KeySerial keySerial);
```

Comment:
We should be consistently using either the parent asset (like here), or the assignment object (like below).

My vote is for the assignment object.